plugins/mac-capture/mac-display-capture.m

```diff
@@ -657,13 +516,8 @@ static bool switch_crop_mode(obs_properties_t *props, obs_property_t *p,
 	.create = display_capture_create,
 	.destroy = display_capture_destroy,
 
-	.output_flags = OBS_SOURCE_VIDEO | OBS_SOURCE_CUSTOM_DRAW |
-			OBS_SOURCE_DO_NOT_DUPLICATE | OBS_SOURCE_SRGB,
-	.video_tick = display_capture_video_tick,
-	.video_render = display_capture_video_render,
-
-	.get_width = display_capture_getwidth,
-	.get_height = display_capture_getheight,
+	.output_flags = OBS_SOURCE_VIDEO | OBS_SOURCE_ASYNC_VIDEO,
```

Comment:
Were the removals of `OBS_SOURCE_DO_NOT_DUPLICATE` and `OBS_SOURCE_SRGB` intentional?  Does the removal of either have any effect?

Reply:
They were intentional, preventing duplication seemed unnecessary and all other plugins provide no color space except one, and its `OBS_SOURCE_CEA_708` not SRGB. It can be added back if desired.

Reply:
Traditionally, we don't allow display sources to be duplicated because creating a bunch of new display sources (which users tend to do when given both options) instead of reusing an existing source has been known to lead to issues in the past, either performance or texture-related.  Whether or not this is still true in this case would probably need some testing.

I'll leave the call on `OBS_SOURCE_SRGB` up to @jpark37 since he knows the SRGB stuff best.

Reply:
You can remove OBS_SOURCE_VIDEO; it's part of OBS_SOURCE_ASYNC_VIDEO (OBS_SOURCE_ASYNC | OBS_SOURCE_VIDEO).

I might change my mind about this later, but it's probably fine to remove OBS_SOURCE_SRGB in this PR for consistency. What the SRGB flag really means is that the plugin has been vetted for SRGB GPU resource usage. We only use the flag to determine if inputs and filters are matched up to skip a draw call, but that optimization is already dead from ASYNC_VIDEO, which is why I didn't bother to update any async input sources.

Also, CEA 708 is about captions as far as I can tell, not color spaces.